### PR TITLE
feat: reposition the action buttons of the section to slide only from…

### PR DESCRIPTION
**Pull request by marianzburlea**
… one side

### CODE REVIEW

**Review by psyhedeliq — APPROVED** [review]
🥇

**Review by marianzburlea — COMMENTED** [review]
nebunie

**Review by BogdanNegura — APPROVED** [review]
💯

File: src/component/course/course.component.jsx [hunk 1]
```diff
@@ -1,30 +1,106 @@
...
+            }]
+          }
+          else {
+            lectureKeyList[lectureContent.section.id] = [{
+              id: lectureId,
+              ...lectureContent
+            }]
+          }
+        })
+      } catch (e) {
```
Comment (QuintyHH) [inline src/component/course/course.component.jsx:50]: Cool, first time I actually see try/catches in javascript :D

**Review by QuintyHH — APPROVED** [review]
💯

**Comment by marianzburlea** [comment]
> 💯

how about line 20 of the `lecture.component.jsx`
?

**Comment by QuintyHH** [comment]
It's an undocumented feature.

File: src/component/section-list/section-list.component.jsx [hunk 2]
```diff
@@ -19,9 +24,29 @@ const SectionList = ({ data = [], handleUpdate, course = {} }) => {
       .catch(message => console.log(`Weird message!`, message))
```
Comment (QuintyHH) [inline src/component/section-list/section-list.component.jsx:24]: Does this belong here?

File: src/component/course-panel/course-panel.component.jsx [hunk 3]
```diff
@@ -14,19 +20,74 @@ const CoursePanel = () => {
...
-      })
-      .catch(message => console.log(`Weird message!`, message))
+    (async () => {
+      try {
+        await db
+          .collection('course')
+          .doc(id)
+          .delete()
+      } catch (message) {
+        console.log(`Weird message!`, message)
```
Comment (QuintyHH) [inline src/component/course-panel/course-panel.component.jsx:88]: Do you want to catch the error with a console log here?

File: src/component/section-list/section-list.component.jsx [hunk 4]
```diff
@@ -19,9 +24,29 @@ const SectionList = ({ data = [], handleUpdate, course = {} }) => {
...
+  const deleteLectureItem = (id, sectionId) => {
+    db
+      .collection('lecture')
+      .doc(id)
+      .delete()
+      .then(aaa => {
+        // console.log(`Item with id: ${id} is no longer with us`, aaa)
+        updateSectionList(removeLectureFromSectionAction(id, sectionId))
+      })
+      .catch(message => console.log(`Weird message!`, message))
```
Comment (QuintyHH) [inline src/component/section-list/section-list.component.jsx:36]: Same as above, catching in a console.log

**Pull request merged**